doc2json/spp2json/spp/spp_client.py

```diff
@@ -7,16 +7,22 @@
 import ntpath
 from typing import List
 
+import requests
 
 class SppClient:
-    def process(self, input: str, output: str):
-        raise NotImplementedError
+    def process(self, infile: str, outfile: str):
+        with open(infile, 'rb') as f_in:
+            files = {"pdf_file": (f_in.name, f_in, "multipart/form-data")}
+            r = requests.post('http://localhost:8080/detect', files=files)
```

Comment:
maybe pick a different port for spp to avoid collision. i think currently, the flask app and grobid run on 8070 and 8080?